pkg/image/controller/signature/signature_import_controller.go

```diff
@@ -106,8 +106,13 @@ func (s *SignatureImportController) work() bool {
 
 	err := s.syncImageSignatures(key.(string))
 	if err != nil {
-		utilruntime.HandleError(fmt.Errorf("error syncing image %s, it will be retried: %v", key.(string), err))
-		s.queue.AddRateLimited(key)
+		if _, ok := err.(GetSignaturesError); !ok {
+			utilruntime.HandleError(fmt.Errorf("error syncing image %s, it will be retried: %v", key.(string), err))
```

Comment:
can we log the error under V(4) or V(5)? Can help debugging.

Reply:
it was already logged when we got the error.

Reply:
glog.V(4).Infof("Failed to get signatures for %v due to: %v", source.Reference(), err)
		return []imageapi.ImageSignature{}, GetSignaturesError{err}